perception/autoware_multi_object_tracker/include/autoware/multi_object_tracker/tracker/motion_model/ctrv_motion_model.hpp

```diff
@@ -45,13 +45,14 @@ class CTRVMotionModel : public MotionModel
   // motion parameters: process noise and motion limits
   struct MotionParams
   {
-    double q_cov_x = 0.025;     // [m^2/s^2] uncertain position in x, 0.5m/s
-    double q_cov_y = 0.025;     // [m^2/s^2] uncertain position in y, 0.5m/s
-    double q_cov_yaw = 0.1218;  // [rad^2/s^2] uncertain yaw angle, 20deg/s
-    double q_cov_vel = 8.6436;  // [m^2/s^4] uncertain velocity, 0.3G m/s^2
-    double q_cov_wz = 0.5236;   // [rad^2/s^4] uncertain yaw rate, 30deg/s^2
-    double max_vel = 2.78;      // [m/s] maximum velocity
-    double max_wz = 0.5236;     // [rad/s] maximum yaw rate, 30deg/s
+    double q_cov_x = 0.025;          // [m^2/s^2] uncertain position in x, 0.5m/s
+    double q_cov_y = 0.025;          // [m^2/s^2] uncertain position in y, 0.5m/s
+    double q_cov_yaw = 0.1218;       // [rad^2/s^2] uncertain yaw angle, 20deg/s
+    double q_cov_vel = 8.6436;       // [m^2/s^4] uncertain velocity, 0.3G m/s^2
+    double q_cov_wz = 0.5236;        // [rad^2/s^4] uncertain yaw rate, 30deg/s^2
+    double max_vel = 2.78;           // [m/s] maximum velocity
+    double max_wz = 0.5236;          // [rad/s] maximum yaw rate, 30deg/s
+    double max_reverse_vel = -1.38;  // [m/s] maximum reverse velocity, -5km/h
```

Comment:
[suggestion for the future]
This value should be negative. If possible add some comment or add validation  of this struct.

Reply:
thank you for your comment. I added a comment to the parameter explanation.